python-package/xgboost/training.py

```diff
@@ -462,7 +462,7 @@ def cv(params, dtrain, num_boost_round=10, nfold=3, stratified=False, folds=None
                                rank=0,
                                evaluation_result_list=res))
         except EarlyStopException as e:
-            for k in results.keys():
+            for k in results:
```

Comment:
Does this change anything related to code logic?

Reply:
No, the change is to comply with the latest pylint guideline (PyCQA/pylint#699). Logic-wise nothing changed.